packages/eslint-plugin/lib/config/prettier.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
     '@shopify/class-property-semi': 'off',
     '@shopify/binary-assignment-parens': 'off',
     'babel/semi': 'off',
+    'prefer-arrow-callback': 'off',
```

Comment:
We are doing this to support `function Foo(){}` over `const` I assume? Or as things well break?

Reply:
The documentation recommended following the defaults which is off, because we have the prettier plugin.